GsaGH/Helpers/Export/Model/Lists.cs

```diff
@@ -1,3 +1,333 @@
-﻿namespace GsaGH.Helpers.Export {
-  internal class Lists { }
+﻿using System;
+using System.Collections.Concurrent;
+using System.Collections.Generic;
+using System.Linq;
+using Grasshopper.Kernel;
+using GsaAPI;
+using GsaGH.Parameters;
+using OasysUnits.Units;
+
+namespace GsaGH.Helpers.Export {
+  internal class Lists {
+    internal static string GetElementList(GsaList list, ref GsaGuidDictionary<EntityList> apiLists,
```

Comment:
too many args

Reply:
which ones can be removed?

Reply:
is it possible to get them from one object?